firefox_ui_harness/runners/update.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, **kwargs):
 
     def duplicate_application(self, application_folder):
         """Creates a copy of the specified binary."""
-        target_folder = os.path.join(self.workspace, 'binary.backup')
+        target_folder = os.path.join(self.workspace_path, 'binary.backup')
```

Comment:
Right but the idea I had here is that we use a single folder under /tmp and have subfolders for the installed binary and the duplicated binary for update tests. So we don't end up with lots of different folders under tmp. But that's just an idea which we don't have to do. Having the correct behavior with --workspace is way more important.